app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
```

Comment:
No need to import ABC
Elf and Dwarf should be inherited from the Player only

Reply:
These are PyCharm's recommendations